TinyLoRa.h

```diff
@@ -37,7 +37,11 @@
 #define TINY_LORA_H
 
 #include <Arduino.h>
-#include <avr/pgmspace.h>
+#if defined(ARDUINO_ARCH_AVR)
+  #include <avr/pgmspace.h>
+#elif defined(ARDUINO_ARCH_ESP32)
+  #include <pgmspace.h>
+#endif
```

Comment:
Sorry, I don't know why is this appearing!

Reply:
@clavisound The `master` branch of this library was updated ahead of the branch you're working on. Please merge the `master` into this branch and it'll resolve.

Reply:
This is from the latest `master`, it allows compatibility w/ the ESP32

Reply:
I know but I was updated my repo before commit.